packages/flutter_blue/example/pubspec.yaml

```
@@ -0,0 +1,20 @@
+name: flutter_blue_tizen_example
+description: Demonstrates how to use the flutter_blue_tizen plugin.
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+dependencies:
```

Please add newlines between sections

```suggestion

dependencies:
```
and sort the dependencies alphabetically. (Is `cupertino_icons` used?)